Content.Shared/Conveyor/ConveyedComponent.cs

```diff
@@ -3,11 +3,15 @@
 namespace Content.Shared.Conveyor;
 
 /// <summary>
-/// Indicates this entity is currently being conveyed.
+/// Indicates this entity is currently contacting a conveyor and will subscribe to events as appropriate.
+/// For entities actively being conveyed see <see cref="ActiveConveyedComponent"/>.
 /// </summary>
-[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[RegisterComponent, NetworkedComponent]
 public sealed partial class ConveyedComponent : Component
 {
-    [ViewVariables, AutoNetworkedField]
-    public List<EntityUid> Colliding = new();
+    // TODO: Delete if pulling gets fixed.
+    /// <summary>
+    /// True if currently conveying.
+    /// </summary>
+    public bool Conveying;
```

Comment:
Has do be a datafield since you are dirtying it.
```suggestion
[DataField, AutoNetworkedField]
    public bool Conveying;
```